website/docs/docs/build/python-models.md

```diff
@@ -16,26 +16,20 @@ We encourage you to:
 - Weigh in on [next steps for Python models, beyond v1.3](https://github.com/dbt-labs/dbt-core/discussions/5742).
 - Join the **#beta-feedback-python-models** channel in the [dbt Community Slack](https://www.getdbt.com/community/join-the-community/).
 
-<VersionBlock firstVersion="1.3">
 In the following article, you'll see a section titled "❓ **dbt questions**." We are excited to release the first set of functionality in v1.3, which will solve real use cases. We also know this is the first step toward a much wider field of possibility. We don't pretend to have all the answers.
```

Comment:
```suggestion

```

Reply:
sugg: 

- appropriate for beta but remove now that feature is GA
- couldn't find where section "❓ **dbt questions** " was

Reply:
The ❓sections are down towards the end of the article

Reply:
ah, i see the callouts now! i was looking for sections like headings